### PR TITLE
Indexed search k8s

### PR DESCRIPTION
**Pull request by uwedeportivo**
dhall diff to base:

```
{ + PersistentVolumes = …
,   Service = { indexed-search = { spec = Some
                                          { clusterIP = - Some …
                                                        + None …
                                          , ports = Some
                                                    [ { name = - None …
                                                               + Some …
                                                      , …
                                                      }
                                                    ]

                                          , …
                                          }
                                 , …
                                 }
              , indexed-search-indexer = { spec = Some
                                                  { clusterIP = - Some …
                                                                + None …
                                                  , ports = Some
                                                            [ { name = - None …
                                                                       + Some …
                                                              , …
                                                              }
                                                            ]

                                                  , …
                                                  }
                                         , …
                                         }
              }
,   StatefulSet = { indexed-search = { spec = Some
                                              { template = { spec = Some
                                                                    { containers = [ { env = - None …
                                                                                             + Some …
                                                                                     , readinessProbe = Some
                                                                                                        { failureThreshold = - Some …
                                                                                                                             + None …
                                                                                                        , httpGet = Some
                                                                                                                    { port =   < …
                                                                                                                               >
                                                                                                                             . - String
                                                                                                                               + Int
                                                                                                                             - "…"
                                                                                                                             + 6070
                                                                                                                    , …
                                                                                                                    }
                                                                                                        , periodSeconds = - Some …
                                                                                                                          + None …
                                                                                                        , …
                                                                                                        }
                                                                                     , …
                                                                                     }
                                                                                   , { env = - None …
                                                                                             + Some …
                                                                                     , …
                                                                                     }
                                                                                   ]

                                                                    , volumes = - Some …
                                                                                + None …
                                                                    , …
                                                                    }
                                                           , …
                                                           }
                                              , volumeClaimTemplates = - Some …
                                                                       + None …
                                              , …
                                              }
                                     , …
                                     }
                  }
}
```

### CODE REVIEW

File: src/k8s/components/indexed-search/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
```
Comment (ggilmore) [inline src/k8s/components/indexed-search/statefulset.yaml:1]: Did you mean to commit this?

**Review by ggilmore — APPROVED** [review]
LGTM once you write the docs

File: src/k8s/components/indexed-search/statefulset.yaml (hunk 1 above)
Comment (uwedeportivo) [reply src/k8s/components/indexed-search/statefulset.yaml:1]: oh no, good catch

Comment (uwedeportivo) [reply src/k8s/components/indexed-search/statefulset.yaml:1]: i use the source YAML files side by side in a split editor when i work on a component

**Pull request merged**